pandas/core/series.py

```diff
@@ -2030,7 +2033,33 @@ def autocorr(self, lag=1):
 
         Returns
         -------
-        autocorr : float
+        float
+            The Pearson correlation between self and self.shift(lag).
+
+        See also
+        --------
+        Series.corr : Compute the correlation between two Series.
+
+        Notes
+        -----
+        return Nan if the computation of the autocorr is not possible.
```

Comment:
Perhaps say something like if the correlation is not defined. Nan should be written as

    `NaN`